packages/osd-opensearch/src/cli_commands/snapshot.js

```diff
@@ -49,6 +49,7 @@ exports.help = (defaults = {}) => {
       -E                Additional key=value settings to pass to OpenSearch
       --download-only   Download the snapshot but don't actually start it
       --ssl             Sets up SSL on OpenSearch
+      --P               OpenSearch plugin artifact URL to install it on the cluster.
```

Comment:
From the PR description it looks like the way to specify multiple plugins is to use the flag multiple times: `yarn opensearch snapshot --version 2.2.0 --P https://repo1.maven.org/maven2/org/opensearch/plugin/opensearch-job-scheduler/2.2.0.0/opensearch-job-scheduler-2.2.0.0.zip --P https://repo1.maven.org/maven2/org/opensearch/plugin/opensearch-anomaly-detection/2.2.0.0/opensearch-anomaly-detection-2.2.0.0.zip`. We might want to mention that in the description. Does this CLI command also have a help command to update?

Reply:
Updated the description and example help command.